### ISSUE TITLE
Issue: Trace your Java Lambda functions with New Relic and OpenTelemetry

### ISSUE THREAD

**Issue by pramodsreek** [newrelic/docs-website#6902]
## How can we make our docs better?

* Is something confusing? **(No)**
* Is something inaccurate or missing? **(Yes)** 

This line doesn't look correct, I had to update it while trying. "AMD64 / X86_64: arn:aws:lambda:{region}:901920570463:layer:aws-otel-java-agent-arm64-ver-1-11-1:1"
It should be "AMD64 / X86_64: arn:aws:lambda:{region}::901920570463:layer:aws-otel-java-agent-amd64-ver-1-11-1:1"

It would be good to update SAM or Cloud formation templates as well, which is the block below that can be copied.

* Were you unable to complete a task? (What task?) **(No)**

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-java/
* **OS**: Unknown
* **Browser**: Unknown
* **Device**: Unknown

**Comment by kolanos** [comment]
Hi @pramodsreek,

Reviewing [this doc](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) from AWS, the ARN is:

```
arn:aws:lambda:\<region>:901920570463:layer:aws-otel-java-agent-amd64-ver-1-11-1:1
```

There does not appear to be an extra `:` between the account ID and region. So this is quite puzzling.

**Comment by pramodsreek** [comment]
Hi @kolanos,

You are right. There is no extra  `:`. Sorry, that was my cut, paste and concat error.  

I have pasted the ARN that was created for Lambda otel layer while testing traces and metrics with New Relic as destination. The Lambda I created was in us-east-1.

`arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-11-1:1`

So, the update required in doc for AMD64 / X86_64 is

`arn:aws:lambda:{region}:901920570463:layer:aws-otel-java-agent-amd64-ver-1-11-1:1`

Thanks!
